### PR TITLE
fix test compat on OrdinaryDiffEq

### PR DESCRIPTION
**Pull request by simonbyrne**
Alternative to #730.

- [ ] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [ ] Unit tests are included OR N/A.
- [ ] Code is exercised in an integration test OR N/A.
- [ ] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: test/Project.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
-OrdinaryDiffEq = "5.64.0"
+OrdinaryDiffEq = "~5.64.0"
```
Comment (charleskawczynski) [inline test/Project.toml:30]: Can we apply this to all the environments?

Comment (simonbyrne) [reply test/Project.toml:30]: interestingly, we're using a later one with examples

**Comment by simonbyrne** [comment]
bors try

File: test/Project.toml (hunk 1 above)
Comment (charleskawczynski) [reply test/Project.toml]: Yeah, that's strange, I thought that what we're doing was incompatible with the latest version

**Pull request merged**